make/docker.mk

```diff
@@ -3,11 +3,12 @@ TARGET_REGISTRY := quay.io
 IMAGE_TAG ?= ${GIT_COMMIT_ID_SHORT}
 IMAGE ?= ${TARGET_REGISTRY}/${QUAY_NAMESPACE}/${GO_PACKAGE_REPO_NAME}:${IMAGE_TAG}
 QUAY_USERNAME ?= ${QUAY_NAMESPACE}
+IMAGE_PLATFORM ?= linux/amd64
 
 .PHONY: docker-image
 ## Build the binary image
 docker-image: build
-	$(Q)docker build -f build/Dockerfile -t ${IMAGE} .
+	$(Q)docker build --platform ${IMAGE_PLATFORM} -f build/Dockerfile -t ${IMAGE} .
```

Comment:
it would be nice to keep these infra-related changes separate. 
since you want to introduce a change in the toolchain-cicd script which requires additional changes in other repositories, then I would expect that you would create a PR for every repo (host-operator, member-operator) separately. 
It makes the review much easier.

Reply:
I've cherry-picked the infra related changes and raised a new PR: https://github.com/codeready-toolchain/host-operator/pull/703 .
So consider this one as very WIP for the work required on Space Counter story.

I was considering to port same infra changes to member-operator as soon as toolchain-cicd scripts are merged and I need to work on member operator repo. But le me know if it could make sense to address and test the new scripts there as well.